static/js/protein/chart.ts

```diff
@@ -102,46 +130,115 @@ const TISSUE_COLOR_DICT = {
   UrinaryBladder: "sandybrown",
   Vagina: "mistyrose",
 };
-// interface for protein page datatypes which return number values
-export interface ProteinNumData {
-  name: string;
-  value: number;
-}
+// tissue specific colors
+const ERROR_BAR_VAR_COLOR = {
+  Pancreas: "peachpuff",
+  Thyroid: "lightcoral",
+  "Whole Blood": "firebrick",
+};
+// tool tip left position
+const TOOL_TIP_LEFT_POSITION = 230;
+// number to select top data points for large data
+const NUM_DATA_POINTS = 10;
+// number to decide the ticks to be displayed
+const NUM_TICKS = 10;
+// graph specific dimensions
+const GRAPH_HEIGHT_LOW = 200;
```

Comment:
nit: when talking about size, we wouldn't really use low or high. maybe you can either use:
- S, M, L, XL or
- SHORT, MED, TALL, XTALL for height & NARROW, MED, WIDE, XWIDE for width